### ISSUE TITLE
Print LoadPath should identify which entries are -R and which are -Q

### ISSUE THREAD

**Issue by jfehrle** [coq/coq#15909]
The description of [`Require`](https://coq.inria.fr/distrib/current/refman/proof-engine/vernacular-commands.html#coq:cmd.From-%E2%80%A6-Require) in the doc abruptly jumps into -R and -Q options halfway through.  Aside from being an incomplete explanation, it's not easy, especially for beginners, to figure out which loadpath has which option.  Either `Print LoadPath` should show this information or, perhaps better, `Require` should give a helpful message such as "Cannot find a physical path bound to logical path foo.  Perhaps you meant `SomePackage.Subdir.foo`?"  And, of course, improve the doc.

@herbelin What do you think?

**Comment by Alizter** [comment]
So `-R` simply does `-Q` for each subdirectory of the one provided. You can see this if you do `coqide -R someDir SomeDir` for someDir with a lot of subdirectories i.e. your home directory. If you then do `Print LoadPath` you will see that it looks like `-Q` was passed for each subdirectory.

Therefore I am not sure this is something we can provide, other than explaining it in the documentation better.

**Comment by jfehrle** [comment]
> I retract that post

~~strikeout type~~ is helpful for retraction :-)

**Comment by Zimmi48** [comment]
It's not clear this will be useful since, as you've noticed, `-Q` is used for anything in `user-contrib/` / `COQPATH` and `-R` is used only for things where it was explicitly specified as part of the `_CoqProject` file of the current project. So, the rule is sufficiently simple for users to know what is `-R` and what is `-Q` (if the doc is sufficiently clear about that).

**Comment by Zimmi48** [comment]
Right. That's one exception for compatibility reasons. And it only applies to the `Coq` logpath but not the `Ltac2` one.

**Comment by jfehrle** [comment]
We should consider whether to deprecate use of `-R`.

**Comment by Zimmi48** [comment]
What people usually seem to think is that it's useful internally but problematic when importing external libraries.
